core/io/resource_saver.cpp

```diff
@@ -120,7 +120,7 @@ Error ResourceSaver::save(const Ref<Resource> &p_resource, const String &p_path,
 
 		String local_path = ProjectSettings::get_singleton()->localize_path(path);
 
-		Ref<Resource> rwcopy = p_resource;
+		const Ref<Resource> &rwcopy = p_resource;
```

Comment:
This one needs investigating, as it doesn't make sense since no copy is made and `p_resource` is used in parallel with it, I'd exclude it and mark it down as a "to investigate"